src/test/java/org/prebid/server/it/InteractiveoffersTest.java

```diff
@@ -0,0 +1,58 @@
+package org.prebid.server.it;
+
+import io.restassured.response.Response;
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalToJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static io.restassured.RestAssured.given;
+import static java.util.Collections.singletonList;
+
+@RunWith(SpringRunner.class)
+public class InteractiveoffersTest extends IntegrationTest {
+
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromInteractiveoffers() throws IOException, JSONException {
+        // given
+        // Interactiveoffers bid response for imp 001
+        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/interactiveoffers-exchange"))
+                .withRequestBody(equalToJson(
+                        jsonFrom("openrtb2/interactiveoffers/test-interactiveoffers-bid-request.json")))
+                .willReturn(aResponse().withBody(
+                        jsonFrom("openrtb2/interactiveoffers/test-interactiveoffers-bid-response.json"))));
+
+        // pre-bid cache
+        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/cache"))
+                .withRequestBody(equalToJson(
+                        jsonFrom("openrtb2/interactiveoffers/test-cache-interactiveoffers-request.json")))
+                .willReturn(aResponse().withBody(
+                        jsonFrom("openrtb2/interactiveoffers/test-cache-matcher-interactiveoffers.json"))));
+
+        // when
+        final Response response = given(SPEC)
+                .header("Referer", "http://www.example.com")
+                .header("X-Forwarded-For", "193.168.244.1")
+                .header("User-Agent", "userAgent")
+                .header("Origin", "http://www.example.com")
+                // this uids cookie value stands for {"uids":{"interactiveoffers":"interactiveoffers-UID"}}
+                .cookie("uids", "eyJ1aWRzIjp7ImFkbWFuIjoiQUQtVUlEIn19")
```

Comment:
This one stands for `{"uids":{"adman":"AD-UID"}}`
Use this resource https://www.base64encode.org/ to create cookie that stands for `{"uids":{"interactiveoffers":"IO-UID"}}`